### ISSUE TITLE
Generates an AttributeError when used in a pyinstaller-packaged application

### ISSUE THREAD

**Issue by WeissRu** [python-cmd2/cmd2#1275]
https://github.com/python-cmd2/cmd2/blob/031832a76b7a9e25d708153085d18d5366ff318d/cmd2/rl_utils.py#L73C6-L73C6

It seems that sys.stdout is None, which is possible according to the python documentation. Using cmd2 version 2.4.3 from pip.

>Under some conditions stdin, stdout and stderr as well as the original values __stdin__, __stdout__ and __stderr__ can be None.
https://docs.python.org/3/library/sys.html?highlight=None#sys.stdout

Part of the exception:
```
 ...
  File "pymobiledevice3\services\afc.py", line 18, in <module>
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 385, in exec_module
  File "cmd2\__init__.py", line 53, in <module>
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 385, in exec_module
  File "cmd2\cmd2.py", line 125, in <module>
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 385, in exec_module
  File "cmd2\rl_utils.py", line 73, in <module>
AttributeError: 'NoneType' object has no attribute 'isatty'

**Comment by anselor** [comment]
Can you provide some context for when this occurs? Are you running on a phone/tablet?

**Comment by WeissRu** [comment]
I am using pyinstaller to package a windowed application on Windows, and import cmd2
python: 3.9.13 pyinstaller: latest cmd2: latest 

minimal example:
test.py
``` python
from cmd2 import Cmd

```
in console
```console
pyinstaller.exe -w -F test.py
.\dist\test.exe
````
Then it will pop up a window showing this exception
```
Traceback (most recent call last):
  File "test.py", line 1, in <module>
    from cmd2 import Cmd
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 385, in exec_module
  File "cmd2\__init__.py", line 53, in <module>
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 385, in exec_module
  File "cmd2\cmd2.py", line 125, in <module>
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 385, in exec_module
  File "cmd2\rl_utils.py", line 73, in <module>
AttributeError: 'NoneType' object has no attribute 'isatty'

**Comment by kmvanbrunt** [comment]
This was fixed in PR #1264 and will be in the next release.
